src/tutorial/counter-app/multiple-files.md

```diff
@@ -48,3 +48,13 @@ fn main() -> Result<()> {
 
 `color_eyre` is an error report handler for colorful, consistent, and well formatted error reports for all kinds of errors.
 ````
+Let's go ahead and declare these files as modules in `src/lib.rs`
+
+
+```rust
+// lib.rs
+
+{{#include ./ratatui-counter-app/src/lib.rs:lib_mods}}
+```
+
+Now we are ready to start refactoring our app.
```

Comment:
I changed the tutorial such that you don't need `./src/lib.rs` anymore. And the recommendation is to add the `mod` directly to `main.rs`. 

I did this for a few reasons:

1. Most command line TUIs are not libraries and are just binary files, so there's no need for a `./src/lib.rs`.
2. Adding a `./src/lib.rs` makes testing slightly more annoying because now some tests will run twice (once when `cargo test` runs for `main.rs` and another when it runs it for `lib.rs`).
3. It's simpler to understand for someone new to rust and they don't have to worry about `lib` vs `bin` etc.

Reply:
Ah ok! That makes a lot of sense

Reply:
I added a new ANCHOR for the mod imports in `main.rs` and reference that here now